common/redshift/Makefile

```diff
@@ -142,6 +142,7 @@ venv3:
 	. $(VENV3_ACTIVATE) && \
 	python -m pip install -U pip -q && \
 	pip install -r $(COMMON_DIR)/redshift/python3_requirements.txt -q && \
+	([ ! -f python3_requirements.txt ] || pip install -r python3_requirements.txt -q 2>/dev/null) && \
```

Comment:
This line is not needed.

Reply:
Let me explain the logic behind this one: s2sphere is imported at the top of the s2Lib `__init__.py`. The `version` integration test imports the `__version__` from `__init__.py` so, when running the integration tests, `s2sphere` has to be installed in the `venv3`

Reply:
I suppose this is why @vdelacruzb put all the imports inside functions in the quadkey RS module